Documents/Technical Specifications/Technical.md

```diff
@@ -0,0 +1,494 @@
+- The main branch
+- MVP Branch
+- Demo branch
+- One for each functionnalities
+
+### Commit
+
+Each commit must be named in English and have a meaningfull name.
+Multiple commits can be done in one branch.
+Commit as much as possible, it will help us to keep track of the changes.
+You can commit not working code, as long as it's not pushed.
+
+### Pull Request
+
+Each pull request must be named in English and have a meaningfull name.
+Each pull request must be reviewed and approved by at least one other person.
+Each pull request must be merged by the person who created the pull request.
+Each pull request must have a description of the changes made.
+
+### Issue
+
+Each issue must be named in English and have a meaningfull name.
+Each issue must be assigned to a person.
+Each issue must have a description of the problem (c.f [Issues](https://github.com/algosup/2022-2023-project-4-game-design-Team-5/issues)).
+
+# Asset
+
+## Environment
+
+The house asset is from the Unreal Engine Marketplace. It is a triplex house with a lot of rooms and a garden. The asset is free to use for non-commercial projects. The asset is available [here](https://www.unrealengine.com/marketplace/en-US/product/big-triplex-house-villa).
+It's already made and we can use it as a base for our game.
+All materials and textures are already made and applied to the asset.
+
+## Character
+
+### Sube
+
+Sube is the main character and is a sugar cube.
+For the MVP, Sube will be a simple cube with a material of sugar applied to it.
+For the final version, Sube will be a 3D model of a sugar cube with a material of sugar applied to it. It will have to be designed by us on Blender.
+
+### NPC
+
+Npc are going to be other sugar cubes that will be used to interact with the player.
+It may be used to give the player a hint or lore about the game.
+For the MVP, Npc will be a simple cube with a material of sugar applied to it.
+For the final version, Npc will be a 3D model of a sugar cube with a material of sugar applied to it. It will have to be designed by us on Blender.
+
+## Plugin
+
+Gravity change plugin is a plugin that allows us to change the gravity of the player. It is available [here](https://www.unrealengine.com/marketplace/en-US/product/directional-planet-gravity). It is free to use for non-commercial projects. It will be used for the sticky functionnality.
+
+# Gameplay
+
+## Objective
+
+The objective of the game is to reach the cup of coffee. The cup of coffee is the end of the game.
+The game will be separated in levels. Each level will move the cup of coffee further away from the player.
+
+## Camera
+
+## Mechanics
+
+Each mechanic will be explained in details here.
+
+### Movement
+
+The player will be able to move on 2 axis (X / Y by default).
+The player will be able to move at a constant speed.
+The player will be able to move in 4 directions (Up / Down / Left / Right).
+Each direction will be mapped to a key.
+
+- Forward: Z
+- Backward: S
+- Left: Q
+- Right: D
```

Comment:
```suggestion
|          | Key | Alternative Key |
|----------|-----|-----------------|
| Forward  | Z   | ↑               |
| Backward | S   | ↓               |
| Left     | Q   | ←               |
| Right    | D   | →               |
```
Nice looking table + add alternative keys for left handed people.

Reply:
Kept the table part
But Arrow might be used in the future for something else

We might use arrows for the second cube. 
We might add arrow as Alternate Key if we don't introduce the split feature